app/component-library/components/Form/HelpText/HelpText.constants.ts

```diff
@@ -26,7 +26,7 @@ export const TEXT_COLOR_BY_HELPTEXT_SEVERITY: TextColorByHelpTextSeverity = {
 
 // Sample consts
 export const SAMPLE_HELPTEXT_TEXT = 'Sample HelpText Text';
```

Comment:
nit: Our content guidelines suggest we use sentence case for everything. We should reenforce this in our documentation even with sample text 
```suggestion
export const SAMPLE_HELPTEXT_TEXT = 'Sample HelpText text';
```